README.md

```diff
@@ -24,7 +24,9 @@ Hearth is a simple starter kit for the Laravel framework. It provides a few thin
 
 ## Installation
 
-You may use Composer to install Hearth into your new Laravel 9 project:
+Hearth requires PHP >= 8 and Laravel >= 9.
```

Comment:
Should this say "Laravel >= 9.20"?

Reply:
No, it's actually `laravel/framework` >= 9.20 ([a dependency of Laravel](https://github.com/laravel/laravel/blob/1e974481ad7cd643aef9ab0483b9afbeb8b14572/composer.json#L10)), but I've added it as a dependency in `composer.json` so that will be enforced by composer.

Reply:
Thanks for clarifying that.